assets/global.scss

```diff
@@ -1,3 +1,10 @@
+@import 'fontsource-noto-sans/index.css';
```

Comment:
Not sure if we should move this to variables as well to keep everything in one place.